package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "etherspot",
-  "version": "1.19.0",
+  "version": "1.19.1",
```

Comment:
1.20.0 ?

Reply:
thought it's a patch-like change

Reply:
I would rather say it's a BC :)

Reply:
No one used it, so technically no harm done